pkg/controller/modeler/modeler.go

```diff
@@ -211,7 +215,7 @@ func (c *Controller) rebuildModel(ctx context.Context, id uint64) error {
 		return fmt.Errorf("failed to digest realm id: %w", err)
 	}
 	key := fmt.Sprintf("realm:quota:%s", dig)
-	if err := c.limiter.Set(ctx, key, uint64(effective), 24*time.Hour); err != nil {
+	if err := c.limiter.Set(ctx, key, uint64(effective), oneWeek); err != nil {
```

Comment:
We're changing this from one-day to one-week. We might want to note in the description. Should `ttl` in realmadmin/settings ref this constant?

Reply:
Oops this shouldn't be in. This is remnant from my debugging... Removed.